templates/RELEASE_NOTES_unified.md.tmpl

```diff
@@ -26,18 +26,22 @@ Upgrade instructions are available for the following components:
 {{- end }}
 {{- end }}
 
-## Changes
-
+# Changes
 The following are changes to the constituent components since the last Conjur
 OSS Suite release:
-{{ range .Components -}}
+{{ range .Components }}
+- [{{ .Repo }}](#{{ replace .Repo "/" "" -1}})
```

Comment:
Just a comment that doesn't need any changes: This seems like it should be a better fit for the funcmap but it's fine as-is.

Reply:
This is in the funcMap

Reply:
I meant the whole logic. `{{ replace .Repo "/" "" -1 }}` -> custom `{{ markdownHeaderLink .Repo }}` where that custom method then does the replace.